### ISSUE TITLE
Plans for India

### ISSUE THREAD

**Issue by ninad458** [grote/Transportr#461]
I don't see any coverage for India. I would like to do that. I would like to start, but I don't have any open source experience. If I have some help, India should be an interesting case to crack.

**Comment by grote** [comment]
Check the homepage for possible ways to get started: https://transportr.app/contribute/#translating-into-your-language

**Comment by ninad458** [comment]
@grote English should be fine for now. Is there nothing else needed? I don't see India in the covered regions.

**Comment by grote** [comment]
Sorry, I don't get your question. Did you read the section on the homepage about "Adding Support For More Regions"?

**Comment by ninad458** [comment]
Yes. It's more like a query. Will it cover automatically if people from that area start using it?

**Comment by grote** [comment]
I don't get would you mean by "cover automatically".

**Comment by n76** [comment]
In order:

- See if all the desired transit information is [known to Navitia](https://navitia.opendatasoft.com/explore/?sort=modified).
- If not in Navitia find where the [General Transit Feed Specification (GTFS)](https://en.wikipedia.org/wiki/General_Transit_Feed_Specification) information can be downloaded for each transit agency that is to be added to Navitia.
- For each feed, you need to find the license and verify it is compatible with Navitia's license requirements.
- Submit the feeds to Navitia.

All of the above requires no programming experience at all. Once Navitia is tracking the transit agencies you need to:

- Add support for the transit agencies' information to the [Public Transport Enabler](https://github.com/schildbach/public-transport-enabler). This requires some programming knowledge but there are easy examples to follow.
- Once support is added to Public Transport Enabler, a pull request for changes to Transportr is required. Again this requires some programming but there are easy examples to follow.

I thought all of the above was pretty evident from the page Torsten linked but apparently not.

If you wish, I can help with the last two programming steps once you do the foot work of assuring the schedule information is in Navitia.

**Comment by grote** [comment]
@n76 thanks a lot for expanding on my explanation! I think this is even a bit more detailed than what is on the homepage. Would be great if you could do a PR to add it there: https://github.com/grote/Transportr-website/blob/master/content/contribute.md#adding-support-for-more-regions-add_region

**Comment by ninad458** [comment]
@n76 that does clarify it, thank you. I'll look into the initial steps.

**Comment by duggalsu** [comment]
AFAIK only Kochi metro feed (https://kochimetro.org/open-data/) and Delhi bus feed are published openly (https://otd.delhi.gov.in)
